osbs/utils/yaml.py

```diff
@@ -0,0 +1,67 @@
+"""
+Copyright (c) 2020 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+
+from __future__ import print_function, absolute_import, unicode_literals
+
+from pkg_resources import resource_stream
+
+import codecs
+import json
+import jsonschema
+import logging
+import yaml
+
+
+logger = logging.getLogger(__name__)
+
+
+def read_yaml_from_file_path(file_path, schema):
+    with open(file_path) as f:
+        yaml_data = f.read()
+    return read_yaml(yaml_data, schema)
+
+
+def read_yaml(yaml_data, schema):
+    """
+    :param yaml_data: string, yaml content
```

Comment:
It would be nice to update the docstrings for both functions to describe the new package param and fix this docsring (add the missing schema param)

Reply:
Fixed.